### PR TITLE
Fix #48 created standard button component and replace some buttons

### LINKED ISSUE #48: Create standard button component

### PR DESCRIPTION
**Pull request by shaunBoahCodes**

### CODE REVIEW

File: src/components/Shared/StandardButton.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<script setup>
+const props = defineProps({
+  type: {
+    type: String,
+    default: 'button'
+  },
+  text: String,
+  render: Function,
+  activated: Boolean,
```
Comment (cweitat) [inline src/components/Shared/StandardButton.vue:9]: Hi, what is activated?

File: src/components/Shared/StandardButton.vue [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+})
+</script>
+
+<template>
+  <button
+    :type="props.type"
+    class="inline-flex items-center rounded-md px-2.5 py-2 text-sm font-medium transition-all"
+  >
+    <component v-if="props.activated" :is="props.activatedRender" class="h-5 mr-1" />
+    <component v-else :is="props.render" class="h-5 mr-1" />
```
Comment (cweitat) [inline src/components/Shared/StandardButton.vue:21]: u can just name props.render to props.icon to be clear

**Review by cweitat — CHANGES_REQUESTED** [review]

File: src/components/Shared/StandardButton.vue (hunk 1 above)
Comment (shaunBoahCodes) [reply src/components/Shared/StandardButton.vue:9]: it is a state where the button is toggled like check-in => checked-in

Comment (cweitat) [reply src/components/Shared/StandardButton.vue:9]: by doing this, the component is not standardised anymore. If want to change the text then pass in new text from parent. and to have the button look as selected, the term is called active. tailwind has the active class css
https://tailwindcss.com/docs/hover-focus-and-other-states

File: src/components/Shared/StandardButton.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const props = defineProps({
+  type: {
+    type: String,
+    default: 'button'
+  },
+  text: String,
+  icon: Function,
+  activated: Boolean,
+  activatedText: String,
+  actiatedIcon: Function,
```
Comment (cweitat) [inline src/components/Shared/StandardButton.vue:11]: typo. activated
and also can set v-if icon exist and other kind of logic

**Review by cweitat — CHANGES_REQUESTED** [review]

File: src/components/Shared/StandardButton.vue (hunk 3 above)
Comment (shaunBoahCodes) [reply src/components/Shared/StandardButton.vue]: activated props removed

**Pull request merged**